jvb/src/main/java/org/jitsi/videobridge/cc/allocation/BandwidthAllocator.java

```diff
@@ -357,6 +357,14 @@ synchronized void update()
                 oversending);
     }
 
+    /**
+     * Query whether this allocator is forwarding a source from a given endpoint, as of its
+     * most recent allocation decision.
+     */
+    public boolean isForwarding(T endpoint)
```

Comment:
should this just take in a `String endpointId` instead of a `T` ?

Reply:
We need to do the `getId()` at some level - I figured I'd push it downward as far as reasonable, but I could pull it upward instead.  Any reason for the higher level?

Reply:
My thought on things like this is to raise it up, such that methods take only what they actually need/use which reduces coupling.  It also makes testing a bit easier.

Reply:
Ok.